README.md

```diff
@@ -1,72 +1,73 @@
-# use-stripe-cart
+# use-shopping-cart
 
-> A Shopping Cart State and Logic for Stripe in React
-
-[![NPM](https://img.shields.io/npm/v/use-stripe-cart.svg)](https://www.npmjs.com/package/use-stripe-cart) [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
+> A React Hook that handles shopping cart state and logic for Stripe.
 
+[![NPM](https://img.shields.io/npm/v/use-shopping-cart.svg)](https://www.npmjs.com/package/use-shopping-cart) [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
 ## Installation
 
 ```bash
-npm install --save use-stripe-cart
+npm install --save use-shopping-cart
 
-or
+# or
 
-yarn add use-stripe-cart
+yarn add use-shopping-cart
 ```
 
 ## Usage
 
-At the root level of your app, wrap your Root app in the `<CartProvider />`.
+### Initialization
+
+At the root level of your application (or the highest point you'll be using Stripe from), wrap your components in a `<CartProvider>`.
 
-Remember to add your public Stripe key, in this example it's added in in the environment `process.env.REACT_APP_STRIPE_API_PUBLIC`.
```

Comment:
This seems to have been removed from the docs but I think this or something similar should be here or in the installation. Something like,


> If using the library with Stripe, the package expects a REACT_APP_STRIPE_API_PUBLIC to be avaiable in your .env file. An example .env file can be found in .env.example.

Reply:
@doingandlearning I converted this line into a comment just above where `loadStripe` is called. I don't believe that having a longer explanation for that part helps at all.

Also, as far as I know, the package does not expect a `REACT_APP_STRIPE_API_PUBLIC` environment variable in a `.env` file as `use-shopping-cart` doesn't read or use any environment variables, however, if someone wishes to they can use `.env` in their own project for this purpose they can but they don't have to. They only need to provide their public Stripe key.

This is the line I have:
```js
// Remember to add your public Stripe key
const stripePromise = loadStripe(process.env.REACT_APP_STRIPE_API_PUBLIC)
```

Let me know if that's sufficient enough to help the developer understand that they need to remember to use their public Stripe key when calling `loadStripe`.

If they don't know how to add environment variables we could provide links to more in-depth tutorials on that matter, if that helps.

Reply:
I think that's. great idea, and adding env variable links sounds great, too.

I'm planning on making a lot of examples/starters and egghead videos to educate people on how to use this library.